client/src/components/CustomSearchBar.js

```diff
@@ -8,19 +8,49 @@ import SearchIcon from "@material-ui/icons/Search";
 import LoginFormStyler from '../utils/LoginFormStyler';
 import axios from 'axios';
 import {useGlobalState} from '../context/GlobalState';
+import ReviewCard from './ReviewCard';
 
 const CustomSearchBar = (props) => {
+  const [globalState, globalSetState] = useGlobalState();
   const [state, setState] = useState({ searchVal: ''});
   const {classes} = props;
   
   const handleSubmit = async (event) => {
-    await axios.get(`${window.location.origin.toString()}/review/searchByReviewText`, {
+    if (state.searchVal === ''){
```

Comment:
We should return all results if no result is found. Eg. If the user queries something that doesn't exist in the text we should still return all reviews for now

Reply:
can u get on a call thx